content/en/references/cross-account-access.md

```diff
@@ -127,6 +127,19 @@ This means that the bucket name has to be globally unique.
 - `ListObjects`
 - `PutObject`
 
+### Secrets Manager
+
+- `GetSecretValue`
+- `PutSecretValue`
+- `DescribeSecret`
+- `UpdateSecret`
+- `DeleteSecret`
+- `RestoreSecret`
+- `ListSecretVersionIds`
+- `GetResourcePolicy`
+- `PutResourcePolicy`
+- `TagResource`
+- `UntagResource`
```

Comment:
Please sort the operations alphabetically

Reply:
thanks for the suggestion. I've updated the ordering.